nmostesting/Config.py

```diff
@@ -97,25 +101,25 @@
 # Test using authorization as per AMWA IS-10 and BCP-003-02
 ENABLE_AUTH = False
 
-# Where the Authorization Server is located on the network. Required when 'ENABLE_AUTH' is True
-# The hostname must match the CN or SAN in the TLS certificate used by the Authorization Server when combined
-# with the DNS_DOMAIN setting
-AUTH_SERVER_HOSTNAME = "auth"
-AUTH_SERVER_IP = "127.0.0.1"
-AUTH_SERVER_PORT = 443
-
-# Which private and public key to use to generate authorization tokens. These must match the keys used by an
-# authorization server on the network to ensure that Nodes trust tokens generated from them. DO NOT use production
-# keys for testing purposes.
-AUTH_TOKEN_PUBKEY = "test_data/BCP00301/ca/intermediate/certs/intermediate.pubkey.pem"
-AUTH_TOKEN_PRIVKEY = "test_data/BCP00301/ca/intermediate/private/intermediate.key.pem"
-
-# Set the contents of the 'iss' key within generated JSON Web Tokens to match what the network authorization server uses
-AUTH_TOKEN_ISSUER = "https://testsuite.nmos.tv"
-
 # The following token is set by the application at runtime and should be left as 'None'
 AUTH_TOKEN = None
 
+# When testing private_key_jwk OAuth client, mock Auth server uses the jwks_uri to locate the client
```

Comment:
At the moment client data is not cached at `/register` (where the `jwks_url` is specified). So if the client is using `private_key_jwt` the `JWKS_URI` must be set up accordingly before the test. 
For now, we would like the mock Auth as stateless as possible. Maybe this can be done in the future.

Reply:
OK, in which case:

```suggestion
# When testing private_key_jwt OAuth client, mock Auth server uses this JWKS_URI to locate the client
```

Or something even more explicit that makes clear the details provided to `/register` are not currently used!